polygon/rest/models/__init__.py

```diff
@@ -1,13 +1,22 @@
-from .common import *
 from .aggs import *
-from .trades import *
-from .quotes import *
-from .markets import *
-from .tickers import *
-from .splits import *
-from .dividends import *
+from .common import *
 from .conditions import *
+from .contracts import *
+from .dividends import *
 from .exchanges import *
-from .snapshot import *
 from .financials import *
-from .contracts import *
+from .indicators import (
+    SMAIndicatorResults,
+    EMAIndicatorResults,
+    RSIIndicatorResults,
+    MACDIndicatorResults,
+    IndicatorValue,
+    MACDIndicatorValue,
+    Underlying as IndicatorUnderlying,
+)
```

Comment:
Why not `import *`?

Reply:
We have to rename Underlying to IndicatorUnderlying. I personally don't love import * in general

Reply:
Oh, why not just rename `Underlying` to `IndicatorUnderlying` then?